package.json

```diff
@@ -2,43 +2,49 @@
   "name": "jupyter-services",
   "version": "0.0.1",
   "description": "Client APIs for the Jupyter services REST APIs",
-  "dependencies": {},
+  "main": "lib/index.js",
+  "dependencies": {
+    "js-logger": "^1.1.1",
+    "phosphor-signaling": "^0.9.0"
+  },
   "devDependencies": {
-    "del": "^0.1.3",
-    "event-stream": "^3.1.7",
+    "dts-bundle": "^0.3.0",
     "expect.js": "^0.3.1",
-    "gulp": "^3.8.9",
-    "gulp-concat": "^2.5.2",
-    "gulp-header": "^1.2.2",
-    "gulp-rename": "^1.2.0",
-    "gulp-stylus": "^2.0.1",
-    "gulp-typedoc": "^1.1.0",
-    "gulp-typescript": "^2.7.8",
-    "gulp-uglify": "^1.2.0",
-    "mocha": "^2.2.5",
-    "mock-socket": "^0.6.0",
-    "nib": "^1.1.0",
     "karma": "^0.12.36",
     "karma-chrome-launcher": "^0.1.12",
-    "karma-firefox-launcher": "^0.1.6",
     "karma-ie-launcher": "^0.2.0",
-    "karma-safari-launcher": "^0.1.1",
     "karma-mocha": "^0.2.0",
     "karma-mocha-reporter": "^1.0.2",
+    "mocha": "^2.2.5",
+    "mock-socket": "^0.6.0",
+    "rimraf": "^2.4.2",
     "sinon": "^1.15.4",
-    "typescript": "Microsoft/TypeScript#release-1.5"
+    "typescript": "^1.5.3"
+  },
+  "scripts": {
+    "clean": "rimraf lib && rimraf test/build/*.js",
+    "build:src": "tsc --project src",
+    "build:dts": "node scripts/dtsbundle.js",
+    "build:test": "tsc --project test",
+    "build": "npm run build:src && npm run build:dts && npm run build:test",
+    "prepublish": "npm run build",
+    "test": "karma start test/karma.conf.js"
   },
-  "scripts": {},
   "repository": {
     "type": "git",
     "url": "https://github.com/jupyter/jupyter-js-services"
   },
   "keywords": [
+    "jupyter",
+    "js",
+    "services",
+    "notebook"
   ],
   "author": "Project Jupyter",
   "license": "BSD-3-Clause",
   "bugs": {
     "url": "https://github.com/jupyter/jupyter-js-services/issues"
   },
+  "main": "lib/index.js",
   "homepage": "https://github.com/jupyter/jupyter-js-services"
 }
```

Comment:
I think you want a Typescript block here, pointing to your main d.ts file generated by dts-bundle.  See https://github.com/basarat/ts-npm-module/blob/master/package.json#L6-L8 .